sale_margin_percentage/views/res_config_settings_views.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_res_sale_margin_config_settings_inherit" model="ir.ui.view">
+        <field name="name">res.sale.margin.percentage.settings</field>
+        <field name="model">res.config.settings</field>
+        <field name="inherit_id" ref="sale.res_config_settings_view_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//div[@data-string='Sales']//div[hasclass('o_settings_container')][2]" position="inside">
+                <div class="col-xs-12 col-md-6 o_setting_box" attrs="{'invisible': [('module_sale_margin' ,'=', False)]}" groups="base.group_no_one">
+                    <div class="o_setting_left_pane"/>
+                    <div class="o_setting_right_pane">
+                        <label for="label_margin_threshold" string="Margin Threshold"/>
```

Comment:
The `for` attribute should point to the involved field:

```suggestion
                        <label for="margin_threshold" string="Margin Threshold"/>
```

Reply:
Change done.